src/vm/mda.cpp

```diff
@@ -1,4017 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Do we have any plans to bring back MDAs?  I've found at least some of them to be useful in the past, e.g. stack imbalance helping to highlight incorrectly written P/Invokes.